plugins/kubernetes-backend/src/cluster-locator/GkeClusterLocator.test.ts

```diff
@@ -18,6 +18,7 @@ import { ANNOTATION_KUBERNETES_AUTH_PROVIDER } from '@backstage/plugin-kubernete
 import '@backstage/backend-common';
 import { ConfigReader, Config } from '@backstage/config';
 import { GkeClusterLocator } from './GkeClusterLocator';
+import { Duration } from 'luxon';
 
 const mockedListClusters = jest.fn();
```

Comment:
If you actually mock the constructor for `ClusterManagerClient`...
```suggestion
import * as container from '@google-cloud/container';

jest.mock('@google-cloud/container', () => {
  return {
    v1: {
      ClusterManagerClient: jest.fn().mockImplementation(() => {
        listClusters: mockedListClusters;
      }),
    },
  };
});
const mockedListClusters = jest.fn();
```